neurodamus/core/nodeset.py

```diff
@@ -160,6 +160,16 @@ def __init__(self, gids=None, gid_info=None):
     offset = property(lambda self: self._offset)
     max_gid = property(lambda self: self._max_gid)
 
+    def __repr__(self):
+        gids = self.gids(raw_gids=True)
+        n = len(gids)
+        return (
+            f"SelectionNodeSet(n={n}, "
+            f"offset={self.offset}, "
+            f"population={self.population_name}, "
+            f"raw gids={self.gids(raw_gids=True)})"
```

Comment:
```suggestion
            f"raw gids={gids})"
```
The value of `gids` has already been assigned above.

Reply:
right. Done